pkgs/by-name/co/configarr/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nodejs,
+  pnpm_10,
+  makeBinaryWrapper,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "configarr";
+  version = "1.13.4";
+
+  src = fetchFromGitHub {
+    owner = "raydak-labs";
+    repo = "configarr";
+    rev = "v1.13.4";
```

Comment:
```suggestion
    tag = "v{finalAttrs.version}";
```

Reply:
Re: finalAttrs, https://github.com/NixOS/nixpkgs/issues/310373.
If I am to use tag, which seems correct, I would prefer
```nix
tag = "v1.13.4";
```